tools/src/main/java/com/powsybl/tools/CommandLineTools.java

```diff
@@ -94,9 +97,21 @@ private static Options getScriptOptions() {
         return options;
     }
 
+    private static Options hideOptions(Options originalOptions, String... hiddenOptions) {
+        Options filteredOptions = new Options();
+        originalOptions.getOptions().stream().filter(o -> {
+            for (String hiddenOption : hiddenOptions) {
+                if (o.getLongOpt().equals(hiddenOption)) {
+                    return false;
+                }
+            }
+            return true;
```

Comment:
An idea to improve readability would be to convert `hiddenOptions` to a `Set<String>` and use the `contains` method.